tools/testsys/Cargo.toml

```
@@ -31,3 +31,4 @@ term_size = "0.3"
 testsys-config = { path = "../testsys-config/", version = "0.1" }
 tokio = { version = "1", features = ["macros", "rt-multi-thread", "fs"] }
 unescape = "0.1"
+url = "2.3.1"
```

Per our dependency management pattern:

```suggestion
url = "2"
```